drift/hsp_service_interface.py

```diff
@@ -6,22 +6,22 @@
 from kerlescan.constants import AUTH_HEADER_NAME, PIT_SVC_ENDPOINT
 
 
-def fetch_pits(pit_ids, service_auth_key, logger):
+def fetch_hsps(hsp_ids, service_auth_key, logger):
```

Comment:
Long names suck but `hsps` is a hard one for me to translate mentally everytime I read a line of code. Could we enumerate it fully or shorten it to something more like `historical_profiles` or `historical_sys_profiles` ?

Reply:
i had `historical_profiles` but wanted to leave the S in there. I'll go w/ `historical_sys_profiles`